src/main/java/seedu/address/model/Planner.java

```diff
@@ -247,15 +247,15 @@ public void setDays(List<Day> days) {
      * The contacts must not already exist in the address book.
      */
     public void addDays(int n) {
-        this.days.addDays(n);
+        this.days.add(n);
     }
 
     /**
      * Removes {@code key} from this {@code Planner}.
      * {@code key} must exist in the address book.
      */
-    public void removeDay(int n) {
-        this.days.removeDay(n);
+    public void removeDay(Day target) {
```

Comment:
Should change back to the following
```suggestion
    public void removeDay(int n) {
```